packages/atomic/src/components/atomic-result-children-template/atomic-result-children-template.tsx

```diff
@@ -4,7 +4,9 @@ import {MapProp} from '../../utils/props-utils';
 import {ResultTemplateCommon} from '../result-templates/result-template-common';
 
 /**
- * TODO:
+ * The `atomic-result-children-template` component determines the format of the query child results, depending on the conditions that are defined for each template. A `template` element must be the child of an `atomic-result-children-template`, and an `atomic-result-children` must be the parent of each `atomic-result-children-template`.
```

Comment:
```suggestion
 * The `atomic-result-children-template` component determines the format of the child results, depending on the conditions that are defined for each template. A `template` element must be the child of an `atomic-result-children-template`, and an `atomic-result-children` must be the parent of each `atomic-result-children-template`.
```
(just because it was unclear whether we were referring to a child of the query or of the result)